src/odemis/util/test.py

```diff
@@ -43,15 +44,36 @@ def setlimits():
 
 def start_backend(config):
     """
-    Start the backend
+    config (str): path to the microscope config file.
+    Start the backend by checking the currently running backend. Basically 2 cases/scenarios:
```

Comment:
Normally in the docstring the description is above the parameters, so I would update this docstring to:
```
"""
Start the backend by checking the currently running backend. Basically 2 cases/scenarios:
        1. If no running backend => run the requested one.
        2. If there is running backend =>
            a. If the running backend is same as requested one => do nothing
            b. If the running backend is different from the requested one => stop the running, and run the requested one.

config (str): path to the microscope config file.
"""
```